File: parantier-front/src/entities/issue/types/issueFigma.ts
export interface IssueFigma {
  id: number;
  issueId: number;
  title: string;
  url: string;
  description?: string;
  orderNum: number;
  createdAt: string;
  updatedAt: string;
}

export interface CreateFigmaRequest {
  title: string;
  url: string;
  description?: string;
  orderNum?: number;
}

export interface UpdateFigmaRequest {
  title: string;
  url: string;
  description?: string;
  orderNum: number;
}

File: parantier-api/src/main/java/com/mapo/palantier/issue/domain/IssueFigma.java
package com.mapo.palantier.issue.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class IssueFigma {
    private Long id;
    private Long issueId;
    private String title;
    private String url;
    private String description;
    private Integer orderNum;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}
